regtests/ww3_ufs1.3/input/ww3_grid_points.inp

```diff
@@ -0,0 +1,728 @@
+$ --------------------------------------------------------------- $
+ 'GLOBAL 1 deg grid lat 85'
+$
+  1.07  0.035  50  36  0.5
+$
+$ Set model flags
+   F T T T T T
+$
+ 1800.0 450.0 900.0 30.0
+$
+&SIN4 BETAMAX = 1.315,
+ TAUWSHELTER = 1.0,
+ SWELLF = 0.798,
+ SWELLF2 = -0.0127,
+ SWELLF3 = 0.0151,
+ SWELLF4 = 100025.0,
+ SWELLF5 = 1.1999,
+ SWELLF7 = 235500.0 /
+&SNL1
+ NLPROP = 2.502E7 /
+&SDS4
+ FXFM3 = 2.501,
+ SDSC2 = -2.1975e-05,
+ SDSCUM = -0.4032,
+ SDSC6 = 0.2978,
+ SDSBR = 0.0009035 /
+&MISC CICE0 = 0.25,
+ CICEN = 0.75,
+ FLAGTR = 4 /
+&SBT1 GAMMA = -0.038 /
+&PRO3 WDTHCG=1.5, WDTHTH=1.5 /
+END OF NAMELISTS
+$
+ 'RECT' T 'SMPL'
```

Comment:
For points we don't need all of this information?  This could be a much simpler file, for example see: 
https://github.com/NOAA-EMC/global-workflow/blob/feature/coupled-crow/parm/wave/ww3_grid.inp.points

unless this is what is used in operations for GEFS (which would surprise me but I did not check).

Reply:
I changed it to the one from https://github.com/NOAA-EMC/global-workflow/blob/feature/coupled-crow/parm/wave/ww3_grid.inp.points